doc/effects.md

```diff
@@ -42,6 +41,7 @@ There are multiple effects provided by Flame, and you can also
 - [`SizeEffect.by`](#sizeeffectby)
 - [`SizeEffect.to`](#sizeeffectto)
 - [`OpacityEffect`](#opacityeffect)
+- [`ColorEffect`](#coloreffect)
```

Comment:
They were ordered alphabetically before, isn't that better, or what is the reason for this change?

Reply:
Huh? They were never alphabetic: for example, Opacity < Size.
They are roughly ordered by category: first, transform effects, then visual effects, then miscellaneous effects. Maybe we should have a two-level list here to make the structure more obvious.

Reply:
Ah alright, they were pretty close to alphabetical so that's why I assumed they were. Your ordering is fine.